Tests/Marketplace/marketplace_constants.py

```diff
@@ -192,6 +192,7 @@ class PackFolders(enum.Enum):
     XSIAM_DASHBOARDS = 'XSIAMDashboards'
     XSIAM_REPORTS = 'XSIAMReports'
     TRIGGERS = 'Triggers'
+    WIZARD = 'Wizards'
```

Comment:
Seems like the standard is:
```suggestion
    WIZARDS = 'Wizards'
```